legislators-current.yaml

```diff
@@ -34606,6 +34608,7 @@
     district: 2
     party: Republican
     phone: 202-225-1621
+    fax: 202-225-2563 
```

Comment:
Tricksy trailing space fixed.

FYI if you ever see this again -- I'd suggest that a trailing space shouldn't throw an error.